kustomize/dev/ethereum/charts/ethereum/templates/statefulset-consensus.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: consensus
+spec:
+  serviceName: "consensus"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: consensus
+  template:
+    metadata:
+      labels:
+        app: consensus
+    spec:
+      containers:
+      - name: consensus
+        image: sigp/lighthouse:latest
+        command: ["/bin/sh"]
+        args: ["-c", "/scripts/start.sh"]
+        ports:
+        - containerPort: 9000
```

Comment:
When i see a value that must match across files. I feel its best to track that in either a const (`_helpers`) or a variable. Just makes it a bit easier to make changes later, also reduces chances for mismatches in the future.